Core/include/Acts/Utilities/Intersection.hpp

```diff
@@ -145,6 +149,14 @@ class ObjectIntersection {
                                const object_t* object, std::uint8_t index = 0)
       : m_intersection(intersection), m_object(object), m_index(index) {}
 
+  /// Invalid object intersection - only holding the object itself
+  ///
+  /// @param object is the object to be instersected
+  constexpr ObjectIntersection(const object_t* object)
+      : m_intersection(Intersection3D::invalid()),
+        m_object(object),
+        m_index(0) {}
```

Comment:
There is an `invalid` factory method which would fit more to express the purpose of the object creation

Reply:
yes, but the invalid does not take a surface, this is the filler method, i.e. you pre-fill your stream with at this moment invalid intersection, but you keep it to hold the surface.

Reply:
Ok I see but the we could have a different overload for `invalid` to keep this consistent. That way we do not introduce another way of creating invalid intersections

Reply:
I changed it such that the `invalid(...)` could also get an object.